pkg/agent/containerd/containerd.go

```diff
@@ -49,6 +49,12 @@ func Run(ctx context.Context, cfg *config.Node) error {
 	}
 
 	template := configToml
+
+	if cfg.ConfigTemplate != "" {
+		fileBytes, _ := ioutil.ReadFile(cfg.ConfigTemplate)
```

Comment:
can you catch the error here please and return err if it can't read the template